### PR TITLE
Added peas option to garden directory

### PR DESCRIPTION
**Pull request by wlongenecker**
Please review this, @casey-collab

### CODE REVIEW

File: english/garden/peas/peas.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+I thought to myself as I looked down at the peas.
```
Comment (casey-collab) [inline english/garden/peas/peas.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wlongenecker** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**